modules/personalize_elements/personalize_elements.module

```diff
@@ -437,25 +437,3 @@ function personalize_element_load($osid) {
   }
   return $os;
 }
-
```

Comment:
Removed personalize_elements_match_page() because it was moved to visitor_actions and renamed to visitor_actions_match_page().